### PR TITLE
DOCS-7862 Quality Gates Documentation Update

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do? What is the motivation?

Prepares Quality Gates for GA by adding screenshots, updating the landing page, the Setup pages, and adding a guide on rule scopes and custom scopes.

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/quality_gates/_index.md [hunk 1]
```diff
@@ -22,35 +25,92 @@ further_reading:
...
+Quality Gates allows you to proactively enhance software quality by setting rules to block substandard code from deployment. You have control over what is merged into the default branch and deployed to production, and can ensure that the code running in production adheres to high quality standards, reducing incidents and minimizing unwanted behaviors.
+
+{{< img src="quality_gates/rule_type.png" alt="A pipeline rule that fails when code coverage for PCT is zero in Quality Gates" style="width:100%" >}}
+
 Use Quality Gates to:
 
-* Create rules that can block your workflows by using data in Datadog.
-* Give your teams control over what code makes it to production and what doesn't.
-* Continually improve code quality with precise enforcement.
+* Create rules that block workflows by using data in Datadog, ensuring that only code that has passed tests affects your production environment.
```
Comment (KickingTheTV) [inline content/en/quality_gates/_index.md:36]:
```suggestion
* Create rules that block workflows using data in Datadog, ensuring that only code that meets your standards end up in production.
```

**Comment by maycmlee** [comment]
Created DOCS-7894

File: content/en/quality_gates/_index.md [hunk 2]
```diff
@@ -22,35 +25,92 @@ further_reading:
...
-{{< callout url="#" btn_hidden="true" >}}
-Quality Gates is in public beta.
-{{< /callout >}}
-
 ## Overview
 
-Quality Gates allows you to gate your workflows based on signals in Datadog. 
+Quality Gates allow you to control software quality by configuring rules to block substandard code from deployment. You have control over what is merged into the default branch and deployed to production, and can ensure that the code running in production adheres to high quality standards, reducing incidents and minimizing unwanted behaviors.
+
+{{< img src="quality_gates/rule_type.png" alt="A pipeline rule that fails when code coverage for PCT is zero in Quality Gates" style="width:100%" >}}
```
Comment (edrevo) [inline content/en/quality_gates/_index.md:32]: I think this screenshot is not a good example. It should fail when the code coverage is "below or equal", not "above or equal"

File: content/en/quality_gates/guide/_index.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+disable_toc: true
+cascade:
+    algolia:
+        rank: 20
+        category: Guide
+        subcategory: Quality Gates Guides
+---
+
+{{< whatsnext desc="Quality Gates Guides:" >}}
+    {{< nextlink href="/quality_gates/guides/understanding_rule_scopes" >}}Understanding How Rule Scopes Work{{< /nextlink >}}
```
Comment (alai97) [inline content/en/quality_gates/guide/_index.md:14]:
```suggestion
    {{< nextlink href="/quality_gates/guide/understanding_rule_scopes" >}}Understanding How Rule Scopes Work{{< /nextlink >}}
```

File: content/en/quality_gates/guide/understanding_rule_scopes.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+1. Enter `example-repository` in the **Repository** field and click **Include**. 
+1. Click **Add Filter** and select **Branch**. 
+1. Enter `main` in the **Branch** field and click **Exclude**.
+
+{{< img src="ci/scope_not_main_example_repository.png" alt="Rule scope for example-repository and not main branch" style="width:90%;">}}
+
+If a rule does not contain a scope, it is evaluated for all values for that scope.
+For example, if a rule does not contain the `repository` scope, it is evaluated for all repositories.
+
+## Add a custom scope
```
Comment (KickingTheTV) [inline content/en/quality_gates/guide/understanding_rule_scopes.md:46]: Just FYI, we're in the process of removing custom scopes from the product - it'll probably be removed in the next month; not sure if we want to remove it from the docs as we push this update out

**Review by KickingTheTV — APPROVED** [review]
LGTM

I still think we need a new section though that acts as an index of all of the rules customers can configure, with a structure resembling something like this:

Test
- new flaky tests
- performance regressions
- code coverage

Pipeline
- custom measures

Static Analysis
- code vulnerability violations
- code quality violations

Software Composition Analysis
- vulnerabilities
- detected licenses

This would help with both discovery and SEO. Otherwise, it would be very difficult to quickly know what kinds of gates are possible.

File: content/en/quality_gates/_index.md [hunk 5]
```diff
@@ -22,35 +25,92 @@ further_reading:
...
 Use Quality Gates to:
 
-* Create rules that can block your workflows by using data in Datadog.
-* Give your teams control over what code makes it to production and what doesn't.
-* Continually improve code quality with precise enforcement.
+* Create rules that block workflows using data in Datadog, ensuring that only code that meets your standards end up in production.
+* Give your organization the ability to decide what code makes it to production, enhancing your deployment discipline and mitigating potential issues in live environments.
+* Continually improve code quality and system performance with precise enforcement and customizable rules.
+
+By integrating Quality Gates [into your CI/CD pipelines][7], you can create a robust framework for maintaining and improving software quality that aligns with your organization's operational goals and business objectives.
```
Comment (aliciascott) [inline content/en/quality_gates/_index.md:40]:
```suggestion
By integrating Quality Gates [into your CI/CD pipelines][7], you can create a robust framework for maintaining and improving software quality that aligns with your organization's operational goals and business objectives. 
```
This link takes me to /monitors/guide/github_gating/ -> is this correct ?

File: content/en/quality_gates/setup/_index.md [hunk 6]
```diff
@@ -11,58 +11,64 @@ further_reading:
...
+1. Navigate to [**Software Delivery** > **Quality Gates** > **Quality Gate Rules**][2] and click **+ New Rule**.
+1. Select a type of rule: `Test`, `Pipeline`, `Static Analysis`, or `Software Composition Analysis`.
+1. Set the rule scope, which defines when the rule should be evaluated, by selecting `Always evaluate` or `Select when to evaluate`. You can add branches or repositories to include or exclude from the rule scope, or add a custom scope.
 
-The following example shows how to create a static analysis rule that fails when there are one or more static analysis violations with "error" status and "security" category being introduced in a specific commit:
+   {{< img src="quality_gates/setup/custom_scope.png" alt="Adding a custom scope to a rule scope in Quality Gates" style="width:80%;">}}
 
-   {{< img src="ci/qg_rule_condition_sa_errors_security_2.png" alt="Rule for static analysis security errors" style="width:80%;">}}
+   You can create a rule that it is evaluated only on specific repositories and branches. To customize the rule scope, click `Select when to evaluate` and specify  the branch or repository that should be included or excluded.
```
Comment (aliciascott) [inline content/en/quality_gates/setup/_index.md:32]:
```suggestion
   You can create a rule that is evaluated only on specific repositories and branches. To customize the rule scope, click `Select when to evaluate` and specify the branch or repository that should be included or excluded.
```

**Review by aliciascott — CHANGES_REQUESTED** [review]
Hi @alai97 just a couple of small things, let me know if you need a re-review if you end up adding more content based on product's suggestions !

File: content/en/quality_gates/guide/understanding_rule_scopes.md (hunk 4 above)
Comment (alai97) [reply content/en/quality_gates/guide/understanding_rule_scopes.md:46]: Ack, we can remove it then!

File: content/en/quality_gates/_index.md (hunk 5 above)
Comment (alai97) [reply content/en/quality_gates/_index.md:40]: Yes, this was originally a guide in the CI Visibility doc set that we migrated to the Monitors doc set!

**Review by KickingTheTV — COMMENTED** [review]
LGTM

**Pull request merged**